### PR TITLE
Added getAliases to indexer

### PR DESCRIPTION
**Pull request by laurensmertens**
Used in the append option for import

### CODE REVIEW

File: Elasticsearch/Indexer.php [hunk 1]
```diff
@@ -95,4 +95,9 @@ private function getClient(): Client
     {
         return $this->factory->fromConfig($this->options);
     }
+
```
Comment (dameert) [inline Elasticsearch/Indexer.php]: Put public functions before private functions

File: Elasticsearch/Indexer.php [hunk 2]
```diff
@@ -95,4 +95,9 @@ private function getClient(): Client
...
+    public function getAliases($params = array()): array
```
Comment (dameert) [inline Elasticsearch/Indexer.php]: Do we want to pass an array with Params or only the index name as a string? Maybe we should convert this to “getAliasByIndexName” to make the call easier?

Comment (laurensmertens) [reply Elasticsearch/Indexer.php]: OK good idea

File: Elasticsearch/Indexer.php (hunk 1 above)
Comment (laurensmertens) [reply Elasticsearch/Indexer.php]: Done

**Pull request merged**